ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -620,6 +627,8 @@ extern "C" {
 	rb_define_private_method(rb_cContext, "eval_unsafe",(VALUE(*)(...))&rb_context_eval_unsafe, 1);
 	rb_define_private_method(rb_cExternalFunction, "notify_v8", (VALUE(*)(...))&rb_external_function_notify_v8, 0);
 	rb_define_alloc_func(rb_cExternalFunction, allocate_external_function);
+    
+    rb_cDateTime = rb_const_get(rb_cObject, rb_intern("DateTime"));
```

Comment:
Adding in this line before creating the context in code doesn't seem to affect the tests at all, I must not be doing it correctly.
`
    Object.send("remove_const", :DateTime)
`
What cases might this even happen? DateTime is part of the standard lib of ruby right? Would someone have to do this on purpose?